taskcluster/translations_taskgraph/transforms/training_continuation.py

```diff
@@ -0,0 +1,94 @@
+from taskgraph.transforms.base import TransformSequence
+from urllib.parse import urljoin
+import os
+
+CONTINUE_TRAINING_ARTIFACTS = (
+    "devset.out",
+    "model.npz",
+    "model.npz.best-bleu-detok.npz",
```

Comment:
It's fine to have this list but there might be more metrics used, for example OPUS-MT use perplexity